### PR TITLE
Added Turkish language

### PR DESCRIPTION
**Pull request by ooploopl-**
Turkish language available. Made all necessary changes by following your guide.

### CODE REVIEW

File: youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+msgid "Successfully downloaded {0} url(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "{0} url(ler) {1} gün {2} saat {3} dakika {4} saniye içinde başarı ile indirildi"
+
+#: mainframe.py:92
+msgid "Downloads completed"
+msgstr "İndirmeler tamamlandı"
+
+#: mainframe.py:93
+msgid "Downloading {0} url(s)"
+msgstr "İnidiriliyor {0} url(ler)"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po]: Do you need the *i* after the *n*? I think *Indiriliyor* is the correct one

File: youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+msgid "Downloads started"
+msgstr "İndirmeler başlatıldı"
+
+#: mainframe.py:99
+msgid "Downloading latest youtube-dl. Please wait..."
+msgstr "youtube-dl'nin en son sürümü indiriliyor. Lütfen bekleyin..."
+
+#: mainframe.py:100
+msgid "Youtube-dl download failed [{0}]"
+msgstr "Youtube-dl indirme başarısız"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po]: You also need the [{0}] it's the place holder for the error message.

File: youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+msgid "Output"
+msgstr "Çıkış"
+
+#: optionsframe.py:47
+msgid "Subtitles"
+msgstr ""
+
+#: optionsframe.py:48
+msgid "Filesystem"
+msgstr "Alt yazı"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po]: Doesn't *altyazi* means subtitles? I believe this should be the translation for line 146

File: youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+msgid "Playlist Start"
+msgstr "Çalma listesini başlat"
+
+#: optionsframe.py:571
+msgid "Playlist Stop"
+msgstr "Çalma listesini durdur"
+
+#: optionsframe.py:572
+msgid "Max Downloads"
+msgstr "Maks İnidirme"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po]: Again, do you need the *i* after the *n*? I believe this should be *Indirme*

File: youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+
+#: optionsframe.py:630
+msgid "Retries"
+msgstr "yeniden denemeler"
```
Comment (MrS0m30n3) [inline youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po]: The first letter should be in upper case.

File: youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po (hunk 1 above)
Comment (ooploopl-) [inline youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po]: Yes. "indiriliyor" is correct one. I'm on mobile right now. Could you please correct the line?

File: youtube_dl_gui/optionsframe.py [hunk 6]
```diff
@@ -1419,7 +1419,8 @@ class LocalizationTab(TabPanel):
         ('de_DE', 'German'),
         ('ar_AR', 'Arabic'),
         ('pt_BR', 'Portuguese'),
-        ('es_MX', 'Spanish')
+        ('es_MX', 'Spanish'),
+	('tr_TR', 'Turkish')
```
Comment (MrS0m30n3) [inline youtube_dl_gui/optionsframe.py]: You should fix the indentation to match the original one

**Comment by MrS0m30n3** [comment]
Hi @ooploopl- 
You should review your changes.
Let me know if any of my notes are wrong.

**Comment by ooploopl-** [comment]
Okay. I will double check everyline but right now i'm long way home with almost no internet connection. When i get home i will open new request without any error.

-----Original Message-----

**Comment by MrS0m30n3** [comment]
I manually merged this one due to some conflicts.
See commit https://github.com/MrS0m30n3/youtube-dl-gui/commit/ee5617c7c2edb76a889bcf57233891eceba383e9.
Thanks for your contribution.

**Pull request closed without merging**